src/components/scenes/TransactionsExportScene.js

```diff
@@ -91,10 +110,14 @@ export class TransactionsExportSceneComponent extends PureComponent<Props, State
   }
 
   exportFile = async () => {
-    const { startDate, endDate } = this.state
+    const { startDate, endDate, isExportQbo, isExportCsv } = this.state
     if (startDate.getTime() > endDate.getTime()) {
       return showError(s.strings.export_transaction_error)
     }
+    if (Platform.OS === 'android' && isExportQbo && isExportCsv) {
+      showError(s.strings.export_transaction_export_error_2)
+      return
+    }
```

Comment:
This should not be needed. The UI should prevent the user from selecting both export types.